test/langtools/jdk/javadoc/doclet/testUseOption/TestUseOption.java

```diff
@@ -191,4 +195,61 @@ public void test3() {
                 """
                     <a href="../C1.html#%3Cinit%3E(unique.UseMe,unique.UseMe)" class="member-name-link">""");
     }
+
+    @Test
+    public void testSuperclassAndInterfaceTypeArgument(Path base) throws IOException {
+        Files.createDirectories(base);
+        Path out = base.resolve("out");
+        Path src = base.resolve("src");
+
+        Files.createDirectories(out);
+        Files.createDirectories(src);
+        Path one = src.resolve("One.java");
+        Files.writeString(one, """
+            public class One {}
+            """);
+        Path two = src.resolve("Two.java");
+        Files.writeString(two, """
+            import java.util.*;
+            public class Two extends ArrayList<One> implements Comparator<One> {
+            }
+            """);
+
+        javadoc(
+                "-use",
+                "-d", out.toString(),
+                one.toString(), two.toString()
+        );
+        checkExit(Exit.OK);
+
+        checkOrder("class-use/One.html", """
```

Comment:
This method call would be easier to read if the text blocks were indented relative to the opening `"""`.  See the other examples earlier in this file and follow the style of those examples.